libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -1626,10 +1633,10 @@ private static (object value, string error) Where(Expression expression, IMemory
 
                         // the local iterator is pushed as one memory layer in the memory stack
                         stackedMemory.Push(new SimpleObjectMemory(local));
-                        var (r, _) = expression.Children[2].TryEvaluate<bool>(stackedMemory);
+                        var (r, _) = expression.Children[2].TryEvaluate(stackedMemory, new Options());
```

Comment:
>Options [](start = 91, length = 7)

Shouldn't this copy?

Reply:
this is updated, maybe it's the cache in broswer